src/libs/ValidationUtils.js

```diff
@@ -135,6 +135,15 @@ function isValidSSNLastFour(ssnLast4) {
     return CONST.REGEX.SSN_LAST_FOUR.test(ssnLast4);
 }
 
+/**
+ *
+ * @param {String} paypalUsername
+ * @returns {Boolean}
+ */
+function isValidPaypalUsername(paypalUsername) {
+    return paypalUsername && CONST.REGEX.PAYPAL_ME_USERNAME.test(paypalUsername);
```

Comment:
A minor change as this condition does not always return boolean. when `paypalUsername` is `''`.

If someone used this in JSX, It will crash the app. Better to convert `paypalUsername` in boolean.

Reply:
Well thinking more about it, should I change it to:
`!paypalUsername || CONST.REGEX.PAYPAL_ME_USERNAME.test(paypalUsername)`

Current behavior to remove the username is deleting the content.

Reply:
Yeah I don't think we should add the `!paypalUsername` since I'm adding a way to delete it right now

Reply:
Thanks for the comment @stitesExpensify

Reply:
@parasharrajat I've updated the Regex instead by changing * to +. Let me know if that's fine.

Reply:
Yup. that's fine.

Reply:
This one is remaining.

Reply:
Done now.